policyengine_us/parameters/gov/states/id/tax/income/main/head_of_household.yaml

```diff
@@ -0,0 +1,35 @@
+description: Idaho provides the following income tax rate for head of household filers.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Idaho income tax rate head of household filers
+  reference:
+    - title: Idaho State Tax Comission - Individual Income Tax Rate Schedule
+      href: https://tax.idaho.gov/taxes/income-tax/individual-income/individual-income-tax-rate-schedule/
+    - title: IDAHO STATUTES || Section 35.01.01.075 - TAX ON INDIVIDUALS, ESTATES, AND TRUSTS   
+      href: https://casetext.com/regulation/idaho-administrative-code/title-idapa-35-tax-commission-state/rule-350101-income-tax-administrative-rules/section-350101075-tax-on-individuals-estates-and-trusts
+    - title: IDAHO STATUTES, TITLE 63, CHAPTER 30, SECTION 63-3024
```

Comment:
I don't understand the discrepancy here. Tweeted for help: https://twitter.com/MaxGhenis/status/1686957793738133504

I also looked through other parts of the legal code, but nothing added color. Have you found anything?

Let's hold off a bit to see if someone responds. Otherwise we'll add a comment noting the discrepancy and file an issue to resolve it.

Reply:
Mystery solved! Please cite the bill from this reply https://twitter.com/WilboBaggins95/status/1686961677265219585